### PR TITLE
[Bug] flinkclousterid cannot be saved in yarn-session mode

### PR DESCRIPTION
**Pull request by xujiangfeng001**
Problem Summary:

### What is changed and how it works?

Proposal: https://github.com/apache/incubator-streampark/issues/1593

Feel free to ping committers for the review!

## Contribution Checklist

  - Make sure that the pull request corresponds to a [GITHUB issue](https://github.com/apache/incubator-streampark/issues/1593).

  - Name the pull request in the form "[Feature] [component] Title of the pull request", where *Feature* can be replaced by `Hotfix`, `Bug`, etc.

  - Minor fixes should be named following this pattern: `[hotfix] [docs] Fix typo in README.md doc`.

### CODE REVIEW

**Comment by xujiangfeng001** [comment]
@wolfboys  I resubmitted a PR and modified it in https://github.com/apache/incubator-streampark/pull/1594 Code conflicts and code writing. Please review again.thank you

**Comment by wolfboys** [comment]
checkstyle failed, You can run `mvn checkstyle:check` to help you fix this issue.

File: streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue [hunk 1]
```diff
@@ -2298,12 +2298,18 @@
 
       handleSubmitCustomJob(values) {
         const options = this.handleFormValue(values)
-        if(values.flinkClusterId){
+        if(values.flinkClusterId) {
```
Comment (wolfboys) [inline streampark-console/streampark-console-webapp/src/views/flink/app/Add.vue]: There should be a space after the if

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**